### PR TITLE
feat: Tmuxinator runs 4 panes tailing each fedimintd log

### PR DESCRIPTION
**Pull request by justinmoon**
Closes https://github.com/fedimint/fedimint/issues/1937

image

### CODE REVIEW

File: .tmuxinator.yml [hunk 1]
```diff
@@ -21,8 +21,14 @@ windows:
             - source scripts/tmux-user-shell.sh
   - federation:
       panes:
-        - federation:
-          - tail -n +0 -F $FM_LOGS_DIR/fedimint-*.log
```
Comment (justinmoon) [inline .tmuxinator.yml:25]: `$FM_LOGS_DIR/fedimint-*.log` was slightly wrong. Logs are in `$FM_LOGS_DIR/fedimintd-*.log`. Missing the `d` in `fedimintd`.

**Review by mxz42 — APPROVED** [review]
nice, thanks :+1:

**Comment by mxz42** [comment]
The distributed key generation log is also mentioned in #1937.
just as remark: it can be found in `cat $FM_LOGS_DIR/fedimint-dev.log`

**Comment by justinmoon** [comment]
> The distributed key generation log is also mentioned in #1937. just as remark: it can be found in `cat $FM_LOGS_DIR/fedimint-dev.log`

Yea so after https://github.com/fedimint/fedimint/pull/1910, the "federation" tmuxinator pane no longer spawn dkg itself. Instead, dkg (along with everything else) get's spawned [here](https://github.com/fedimint/fedimint/blob/fd4ca76307ed3dcb415490db3b94f67b6930b64d/scripts/tmuxinator.sh#L25) before tmux itself is every launched. It runs in background and logs to file.

So it's a little weird that the DKG logs don't show up visually in tmuxinator anymore. But this should hopefully never fail in tmuxinator. Perhaps we could have each DKG instance log to `fedimintd-<n>.log` like `fedimintd` does? I think that would be an improvement.

**Review by Maan2003 — APPROVED** [review]

**Pull request merged**